extra_types.go

```diff
@@ -176,6 +179,32 @@ func (p PlotPoint) toC() C.ImPlotPoint {
 	return C.ImPlotPoint{x: C.double(p.X), y: C.double(p.Y)}
 }
 
+type PlotTime struct {
+	S  int // second part
+	Us int // microsecond part
+}
```

Comment:
what about making it just `type PlotTime time.Time`? It'll enable type-casting so from `time.Time` to `PlotTime` and the usage'd be easier imao

Reply:
PlotTime has no time zone offset, but `time.Time` has.
PlotTime has lower precision (microseconds vs nanoseconds at `time.Time`) 
It all can cause misunderstand how to use it.
Also no need to make arithmetic operations on every C call at current implementation.

Reply:
ok, sounds fair.